extensions/wikia/AdEngine/js/video/uapVideo.js

```diff
@@ -170,8 +168,16 @@ define('ext.wikia.adEngine.video.uapVideo', [
 			win.addEventListener('resize', throttle(resizeVideo));
 			video.addEventListener('play', resizeVideo);
 
-			return video;
-		});
+			if (typeof onLoad === 'function') {
+				onLoad(video);
+			}
+		}
+
+		if (params.player === 'playwire') {
+			loadPlaywire(params, videoContainer, providerContainer, videoSettings, onPlayerLoad);
+		} else {
+			loadPorvata(params, videoContainer, providerContainer, videoSettings, onPlayerLoad);
+		}
```

Comment:
I can't find this file being used. Maybe we can simply remove it?

Reply:
I didn't check it. +1

Reply:
All usages were removed in: https://github.com/Wikia/app/commit/fd00549aca316105605475346d862fa005db9318